samples/name_domains_values/Coded_Domain_HostileOrEnemy.csv

```diff
@@ -0,0 +1,2 @@
+Name,Value
+Hostile (Enemy),ENY
```

Comment:
I don't recall this being in the list from #200 or know how this 1-value domain will be used, but be aware that if this file is in the domains folder, a domain will be created for this in the gdb.